### DISCUSSION TITLE
Web-based Modpack Maker

### DISCUSSION THREAD

**Discussion by cootshk** [modrinth/.github#127] [Feature requests]
Similar to what is in the current (alpha) launcher when creating an instance

You can press create modpack, have a search bar that shows mods that are compatible with your game version and loader (Also selecting Quilt should also show fabric mods)

Auto-install dependencies (pulled from the mod version page)

Once you are done, you can add an icon/description/all of that other stuff

**Comment by cootshk** [comment]
# You can rename many of these to say modpack
![image](https://github.com/modrinth/.github/assets/83678457/6999209b-da31-4268-95f8-59c576990518)
![image](https://github.com/modrinth/.github/assets/83678457/27d7d8d2-8fa4-48b5-b6ef-c506461fbf7c)
Selecting Quilt should also show fabric mods
![image](https://github.com/modrinth/.github/assets/83678457/15ba9098-5c44-46e1-9232-8e0fafcc993d)
Would auto-install dependencies and have a pop-up to install optional dependencies (with a don't show again button)

**Comment by Madis0** [comment]
Not a good idea. Modpacks are more than just a list of mods; creating a modpack based on a list alone has a good chance that your mods just don't work the way you'd expect, even if all dependencies are downloaded. You need to play through and test the instance properly.

**Reply by VELD-Dev** [reply]
Yes but this is using additionnal storage when optimizations could be done by letting modpackers, after having tried their modpack, list the mods they used in a proper way. Actually this could also prevent from endorsing some creators preferences who don't want their mod to be into modpacks. It can also prevent mods repost, which is a big thing actually. I'd like to relaunch this thread because modpack creation just feels like it was put at some moment and never touched ever again, like "nah anyway nobody will ever use it".
